package.json

```diff
@@ -34,21 +34,21 @@
     "test:watch": "vitest watch"
   },
   "dependencies": {
-    "@nuxt/kit": "^3.9.0",
-    "@nuxtjs/tailwindcss": "^6.10.1",
-    "h3-clerk": "^0.3.8",
-    "vue-clerk": "^0.1.1"
+    "@nuxt/kit": "^3.12.1",
+    "@nuxtjs/tailwindcss": "^6.12.0",
```

Comment:
https://github.com/RodrigoProjects/nuxt-clerk/issues/4 - I agree with this. Do we really need tailwind as dep?